types/request/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
```

Comment:
do not use `pacakge.json` dependencies for packages that already exist on DT. tough-cookie already exists see: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/tough-cookie/index.d.ts . just add the import and the tsconfig.json should resolve it correctly.